packages/block-library/src/legacy-widget/index.php

```diff
@@ -60,14 +105,20 @@ function register_block_core_legacy_widget() {
 		'core/legacy-widget',
 		array(
 			'attributes'      => array(
-				'identifier'       => array(
+				'widgetClass'  => array(
 					'type' => 'string',
 				),
-				'instance'         => array(
-					'type' => 'object',
+				'widgetId'     => array(
+					'type' => 'string',
 				),
-				'isCallbackWidget' => array(
-					'type' => 'boolean',
+				'idBase'       => array(
+					'type' => 'string',
+				),
+				'widgetNumber' => array(
```

Comment:
nit: Is it redundant to prefix all of these attributes with `widget`? The block is called a `legacy-widget`, after all.